services/src/test/java/org/keycloak/broker/oidc/mappers/AdvancedClaimToRoleMapperTest.java

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
This should be converted to a functional tests. See testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/OidcUserAttributeMapperTest.java as an example.

Reply:
Okay, I'll convert the test, thanks for the hint to a similar test.

Reply:
done